tests/test_playbooks/auth_source_ldap.yml

```
@@ -73,7 +73,7 @@
       vars:
         location_name: "{{ item }}"
         location_state: "absent"
-      with_items: "{{ domain_locations }}"
+      with_items: "{{ domain_locations | reverse }}"
```

```suggestion
      with_items: "{{ domain_locations[::-1] }}"
```
`reverse` doesn't seem to work correctly. The test was incorrectly passing and still tried to delete the parent location first on the Satellite. Found this syntax online and it seems to be working correctly.

---

Oooh, I know why...

Can you try `| reverse | list`? Reverse probably returns a generator by default 🙄

---

Yeah that works!